tests/integration.rs

```diff
@@ -736,3 +736,53 @@ fn test_expire() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_alloc_data_type() -> Result<()> {
```

Comment:
Thank you for adding a test for this Module.

To test the new functionality (`digest`), we can use the `DEBUG` command using `redis::cmd("DEBUG")` on the module data type key and validate that it returns a value without raising any error. This will validate your new callback is being invoked

Reply:
Thanks for the feedback, updated